Packages/com.unity.inputsystem/InputSystem/Plugins/UI/MouseModel.cs

```diff
@@ -0,0 +1,317 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace UnityEngine.Experimental.Input.Plugins.UI
+{
+    /// <summary>
+    /// A series of flags to determine if a button has been pressed or released since the last time checked.
+    /// Useful for identifying press/release events that occur in a single frame or sample.
+    /// </summary>
+    [Flags]
+    public enum ButtonDeltaState
```

Comment:
What's the case for exposing the APIs in here? Seems like everything in here should be internal and the input modules being the only thing that the plugin exports.